arrow/src/compute/util.rs

```diff
@@ -206,25 +212,49 @@ pub(super) mod tests {
             make_data_with_null_bit_buffer(8, 0, Some(Buffer::from([0b01001010])));
         let inverse_bitmap =
             make_data_with_null_bit_buffer(8, 0, Some(Buffer::from([0b10110101])));
+        let some_other_bitmap =
+            make_data_with_null_bit_buffer(8, 0, Some(Buffer::from([0b11010111])));
+        assert!(combine_option_bitmap(&[], 8).is_err());
```

Comment:
nit: test more precisely:
```suggestion
        assert_eq!(
            combine_option_bitmap(&[], 8).unwrap_err().to_string(),
            "Compute error: Arrays must not be empty",
        );
```